README.md

```diff
@@ -136,6 +136,7 @@ in February 2016.
 |[Siegen][siegen-wikipedia]|[City of Siegen][siegen-markets]|
 |[Solingen][solingen-wikipedia]|[City of Solingen][solingen-markets]|
 |[Stralsund][stralsund-wikipedia]|[City of Stralsund][stralsund-markets]|
+|[Stuttgart][stuttgart-wikipedia]|[Stuttgarter Wochenmärkte][stuttgart-markets]|
```

Comment:
Please check the preview of the README on your branch [here](https://github.com/hbruch/wo-ist-markt.github.io/tree/stuttgart) - you forgot to add the links.

Reply:
Added the links with commit  107229e